tests/conftest.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+from promdens.promdens import LaserPulse
+
+@pytest.fixture
+def make_pulse():
+
+    def _make_pulse(omega=0.1, fwhm=1.0, lchirp=0, t0=0, envelope_type="gauss"):
```

Comment:
Just a super simple helper function with some default values to make creating LaserPulse instance less verbose in tests.